### PR TITLE
Fixes issues with registries that have images in new docker 1.10 mani…

### PR DESCRIPTION
**Pull request by rqc**
…fest.

Images prior to docker 1.10 seem to be unsupported by registry:v2 and artifactory.
When uploading images that haven't been migrated to the new format it was causing issues in the parsing.
Added new UI fields and error handling.
Removed delete action from the UI for repositories since it does not work at the moment. Need to re-implement as deletion of manifest for an image.
**Needs more testing**

UI changes:
image
image
image
image

### CODE REVIEW

**Comment by rqc** [comment]
@JonathanRosado could you please take a look at the UI part of this to make sure I didn't miss anything. Thanks!

File: controller/static/app/registry/registry.service.js [hunk 1]
```diff
@@ -24,9 +24,9 @@
                     });
                 return promise;
             },
-            inspectRepository: function(registryName, repositoryName) {
+            inspectRepository: function(registryId, repositoryName, repositoryTag) {
```
Comment (JonathanRosado) [inline controller/static/app/registry/registry.service.js]: @rqc Change `listRepositories`'s `name` parameter to is as well

File: controller/static/app/registry/repository.html [hunk 2]
```diff
@@ -2,72 +2,63 @@
...
-                        <div class="ui statistic">
-                            <div class="value">
-                                <i class="disk outline icon"></i> {{vm.selectedRepository.size / 1048576 | number:2}}
-                            </div>
-                            <div class="label">
-                                MB
-                            </div>
-                        </div>
-                    </div>
+                    <!--<div class="column">-->
```
Comment (JonathanRosado) [inline controller/static/app/registry/repository.html]: @rqc Do we need all these comments?

File: controller/static/app/registry/registry.service.js (hunk 1 above)
Comment (rqc) [inline controller/static/app/registry/registry.service.js]: could you point me to the line please?

**Comment by rqc** [comment]
@JonathanRosado indeed, we need to update the Projects UI (in the add image, edit image areas) to use the associated registry id to request repositories, and then later to store that information in the backend to associate the image with the registry. Right now, I see that you send a `image.registry` field, but that is not implemented in the backend. We need to also add some support for that. Do you care to take a stab at this is @soninob approves? You can also make the tweaks in the backend. 

@soninob This is really important since this was functionality that we had before and it broke after docker 1.10. Thanks!

**Comment by rqc** [comment]
Opened a new issue for the regressions: https://github.com/eedevops/shipyard/issues/64

**Pull request merged**